mkosi.conf.d/20-serpentos.conf

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=serpentos
+
+[Content]
+ShimBootloader=none
+Packages=
+Packages=
+        systemd
+        dash
```

Comment:
dash is useful for scripting but pretty useless as an emergency login shell so we should install bash as well